src/Stratis.Bitcoin.Features.Apps/AppsStore.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Logging;
+using Stratis.Bitcoin.Features.Apps.Interfaces;
+using Stratis.Bitcoin.Utilities.Extensions;
+
+namespace Stratis.Bitcoin.Features.Apps
+{
+    public class AppsStore : IAppsStore
+    {
+        private readonly ILogger logger;
+        private readonly List<IStratisApp> applications = new List<IStratisApp>();
+        private readonly IAppsFileService appsFileService;
+        private readonly IStratisAppFactory appFactory;
+
+        public AppsStore(ILoggerFactory loggerFactory, IAppsFileService appsFileService, IStratisAppFactory appFactory)
+        {
+            this.appsFileService = appsFileService;
+            this.appFactory = appFactory;
+            this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
+
+            this.Load();
+        }
+
+        public IEnumerable<IStratisApp> Applications => this.applications;
+
+        private void Load()
+        {
+            IEnumerable<FileInfo> configs = this.appsFileService.GetStratisAppConfigFileInfos();
+            var apps = configs.Select(this.CreateApp).Where(x => x != null).ToList();
+
+            if (apps.IsEmpty())
+            {
+                this.logger.LogWarning(
+                    $"No Stratis applications found at or below {this.appsFileService.StratisAppsFolderPath}");
+                return;
+            }
+
+            this.applications.AddRange(apps);
+        }
+
+        private IStratisApp CreateApp(FileInfo fileInfo)
```

Comment:
`CreateApp` sounds like you're creating an app rather than loading one.
Would `LoadApp` be more appropriate?

Reply:
It's a private method that creates an instance of a StratisApp per app in config, ready for hosting.  No loading involved.